app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
It is recommended to use the Exception as the parent class for custom exceptions, not the BaseException.